diesel/src/lib.rs

```diff
@@ -87,15 +87,83 @@
 //!
 //! ## Getting help
 //!
-//! If you run into problems, Diesel has a very active Gitter room.
-//! You can come ask for help at
+//! If you run into problems, Diesel has an active community.
+//! Either open a new [discussion] thread at diesel github repository or
+//! use the active Gitter room at
 //! [gitter.im/diesel-rs/diesel](https://gitter.im/diesel-rs/diesel)
+//!
+//! [discussion]: https://github.com/diesel-rs/diesel/discussions/categories/q-a
+//!
+//! # Crate feature flags
+//!
+//! The following feature flags are considered to be part of diesels public
+//! API. Any feature flag that is not listed here is **not** considered to
+//! be part of the public API and can disappear at any point in time:
+
+//!
+//! - `sqlite`: This feature enables the diesel sqlite backend. Enabling this feature requires a compatible copy
```

Comment:
I think it would be useful to document the `returning_clauses_for_sqlite_3_35` feature. I spent some time trying to figure out why they weren't working for me on master the other day before discovering that support was locked behind the said feature.